components/support/sync15-traits/src/server_timestamp.rs

```diff
@@ -90,8 +90,8 @@ impl<'de> serde::de::Visitor<'de> for TimestampVisitor {
 }
 
 impl<'de> serde::de::Deserialize<'de> for ServerTimestamp {
```

Comment:
Sorry, one more thing—would you mind deleting `TimestampVisitor`, since we've replaced it with `f64::deserialize`? Thanks!

Reply:
Yeah np!